File: ConverterToolTests/SolutionFiles/JavaFiles/InterfaceMethods.java
import java.io.*;
import java.lang.*;
import java.util.*;

interface InterfaceMethods {

    //  wheel revolutions per minute
    void changeCadence(int newValue);

    void changeGear(int newValue);

    void speedUp(int increment);

    void applyBrakes(int decrement);
}

File: ConverterToolTests/SolutionFiles/CsharpFiles/InterfaceMethods.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConverterToolTests.SolutionFiles.CsharpFiles
{
    interface IInterfaceMethods
    {
        void ChangeCadence(int newValue);

        void ChangeGear(int newValue);

        void SpeedUp(int increment);

        void ApplyBrakes(int decrement);
    }
}
